go.mod

```diff
@@ -1,18 +1,20 @@
 module github.com/openshift/ibm-powervs-block-csi-driver-operator
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.3
```

Comment:
SAST scanner used to fail with this line. Do you think we can remove this and update go version at line 3?
ref: https://github.com/openshift/ibm-powervs-block-csi-driver-operator/pull/68